test/carve/main_test.clj

```diff
@@ -62,6 +64,11 @@ test-resources/app/app.clj:9:1 app/ignore-me")
     (is (thrown? clojure.lang.ExceptionInfo
                  (run-main "{:paths [src test]}"))))
 
+  (testing "Generate random options validate"
+    (doseq [o (g/sample (s/gen ::main/opts))]
+      ;; the paths needs to exist to to simplify the test we just hard code it
+      (is nil? (main/validate-opts! (assoc o :paths ["."])))))
```

Comment:
a bit weird to check nil? but well validate-opts! would simply throw an exception if it fails

Reply:
`(is nil? ...)` always succeeds, you should write `(is (nil? ...))` or `(is (empty?  ...))`

Reply:
Ah yes of course, weirdly kaocha didn't notice that even in a test by itself, maybe it only checks the `=/not=` cases.
Anyway fixed it thanks!